src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1203,7 +1203,7 @@ ss::future<add_offsets_tx_reply> tx_gateway_frontend::do_add_offsets_to_tx(
               .error_code = tx_errc::not_coordinator};
         }
         co_return add_offsets_tx_reply{
-          .error_code = tx_errc::invalid_txn_state};
+          .error_code = tx_errc::coordinator_not_available};
```

Comment:
Is `coordinator_not_available` retryable ? It seems that if sync failed we may want client to retry ?

Reply:
yes, it's retryable

```
 * In the context of the transactional coordinator, this error will be returned if the underlying transactional log
 * is under replicated or if an append to the log times out.
 */
public class CoordinatorNotAvailableException extends RetriableException {
    public static final CoordinatorNotAvailableException INSTANCE = new CoordinatorNotAvailableException();
```